llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Fusion.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright 2021 The IREE Authors
```

Comment:
Nit: this should be 2022

Reply:
> This seems similar to the other PR. Is there a dependence here?

This one is stacked on top of the other one and adds a second op with a different type of user control for fusion.

Reply:
Fixed, thanks!